### ISSUE TITLE
[Bug] - tables not created in database, no duplicates found

### ISSUE THREAD

**Issue by invario** [eldertek/duplicatefinder#60]
**Describe the bug**
No duplicates are found after installing the app.  From a command prompt, I executed the following:

```
occ duplicates:find-all
Start searching files for USERNAME in path /USERNAME/files

In DbalException.php line 71:

  An exception occurred while executing a query: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'nextcloud.oc_duplicatef
  inder_finfo' doesn't exist


In ExceptionConverter.php line 49:

  An exception occurred while executing a query: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'nextcloud.oc_duplicatef
  inder_finfo' doesn't exist


In Exception.php line 28:

  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'nextcloud.oc_duplicatefinder_finfo' doesn't exist


In Statement.php line 130:

  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'nextcloud.oc_duplicatefinder_finfo' doesn't exist


duplicates:find-all [-u|--user USER] [-p|--path PATH]

```

**To Reproduce**
1. Install Duplicate Finder
2. Use it

I took the following troubleshooting steps:
1. Removed Duplicate Finder
2. Ensured the duplicatefinder directory was completed gone from nextcloud/apps/
3. Logged into MariaDB to verify that the oc_duplicatefinder doesn't exist in my nextcloud database
4. Reinstalled Duplicate Finder
5. Re-ran occ duplicates:find-all and also tried occ duplicates:clear. 
6. Both return the error previously mentioned
7. Logged into MariaDB nextcloud database. There is no oc_duplicatefinder table.

**Versions**
- Nextcloud 28.0.3
- Duplicate Finder 1.1.8
- PHP 8.2.14
- FreeBSD 13.0-RELEASE inside iocage jail under TrueNAS Core 13

Is something preventing DuplicateFinder from creating the required tables?  I think a year or two ago I had an older version of Duplicate Finder installed; is it possible some old remnants remain somewhere?

Thank yoiu.

**Comment by invario** [comment]
Well, that was painful, but after poring over my NC database, I scrubbed every mention of duplicatefinder I could find.

The following tables were affected:
- oc_filecache (a few rows here)
- oc_appconfig (a few rows here)
- oc_migration (**many** rows in here)

After manually deleting all rows mentioning duplicatefinder from the above tables, I reinstalled duplicatefinder and ran "occ duplicate:find-all"  and it appears to be doing its thing. Will update with results!
